pkg/policyengine/PolicyDispatcher.go

```diff
@@ -36,8 +39,27 @@ const (
 var plog = logrus.WithField("component", "PolicyEngine")
 var MyPolicyHandler PolicyHandler
 
+type PolicyDecider interface {
+	AddLBPolicy(lbPolicy *LBPolicy) error
+	DeleteLBPolicy(lbPolicy *LBPolicy) error
+
+	AddAccessPolicy(policy *api.Policy) error
+	DeleteAccessPolicy(policy *api.Policy) error
+
+	AuthorizeAndRouteConnection(connReq *event.ConnectionRequestAttr) (event.ConnectionRequestResp, error)
+
+	AddPeer(peer *api.Peer) error
+	DeletePeer(name string) error
+
+	AddBinding(imp *api.Binding) (event.Action, error)
```

Comment:
Currently this always returns "allow". Same for `AddExport`.
Do we plan to support policies disallowing this kind of operations?
@elevran @kfirtoledo

Reply:
(this is something that was possible in the old policy engine, but is currently not implemented in the new policy engine)